src/TestInfrastructure/Framework/ConfigurableVsInfoBarUIElement.cs

```diff
@@ -132,4 +133,19 @@ public IVsInfoBar Model
 
         #endregion Test helpers
     }
-}
\ No newline at end of file
+
+    public class ConfigurableVsInfoBarActionItem : IVsInfoBarActionItem
```

Comment:
For some reason I'm getting failures on 2019 when I try to mock IVsInfoBarActionItem:
```
Test method SonarLint.VisualStudio.Integration.UnitTests.InfoBarManagerTests.InfoBarManager_AttachInfoBar threw exception:
System.MissingMethodException: Error: Missing method 'instance string [SonarLint.2019] Microsoft.VisualStudio.Shell.Interop.IVsInfoBarActionItem::get_Text()' from class 'Castle.Proxies.IVsInfoBarActionItemProxy'.
```

Reply:
That's weird; `Castles.Proxies` is part of the Mock framework isn't it? Why is the Mock framework involved?

Reply:
I don't know and I still get failures on 2019 :(